rfcs/text/0012_background_sessions.md

```diff
@@ -0,0 +1,291 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
```

Comment:
__quick note:__ you may want to increment RFC number (`12` --> `13`) since `12` [is occupied now](https://github.com/elastic/kibana/blob/master/rfcs/text/0012_encryption_key_rotation.md).